handout/components/creating_components.md

```diff
@@ -1,6 +1,6 @@
 # Creating Components
 
-Components in Angular 2 build upon the lessons learned from Angular 1.5. We define a component's application logic inside a class. To this we attach `@Component`, a TypeScript `decorator`, which allows you to modify a class or function definition and adds metadata to properties and function arguments.
```

Comment:
Again, in this case use Angular 2 and Angular 1.5 because we are comparing two specific versions.

Also, I think the premise of this paragraph is wrong. I'm pretty sure that the `.component` function of Angular 1.5 is a backport from Angular 2 to ease the migration. It's worth investigating that and creating a separate PR if the paragraph needs to be updated.